ingestion/src/metadata/cmd.py

```diff
@@ -130,88 +131,93 @@ def report(config: str) -> None:
         ) from exc
 
 
+# metadata docker --start
+# metadata docker --start -f <path_to_docker_compose_file>
+# metadata docker --stop
+# metadata docker --pause
+# metadata docker --resume
+
+
 @metadata.command()
 @click.option("--start", help="Start release Docker containers", is_flag=True)
-@click.option("--stop", help="Stop Docker containers (local and release)", is_flag=True)
 @click.option(
```

Comment:
keeping the clean to remove the images and prune containers would be better

Reply:
Added the --clean option which will remove the images, prune containers, remove networks and volumes and delete the temporary downloaded docker-compose.yml file.